wagtail/rich_text/feature_registry.py

```diff
@@ -98,6 +103,18 @@ def get_converter_rule(self, converter_name, feature_name):
         except KeyError:
             return None
 
+    def register_frontend_rewriter(self, rewriter, order=300):
```

Comment:
I don’t get why we’d want a default value?

Reply:
I think I mainly did this on account of cargo-cult, in that most of the similarly ordered objects, like `MenuItem`s and `hooks.register` did have a sensible default - I'm not precious either way though